js/index.js

```diff
@@ -38,5 +38,101 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
+// Images - Logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Images - CTA
+let heroImg = document.getElementById('cta-img');
+heroImg.setAttribute('src', siteContent['cta']['img-src']);
+
+// Images - Middle
+
```

Comment:
To stay consistent with the other comments this new line on line 51 should be removed as all other comments do not have two new lines before the code.